ts/Extensions/Annotations/Types/InfinityLine.ts

```diff
@@ -227,6 +209,30 @@ InfinityLine.prototype.defaultOptions = merge(
  * @apioption annotations.infinityLine
  */
 
```

Comment:
It looks like `@apioption` doclets have to be move to the end of the file caused by changed code structure:
```ts
export default …

/**
 * …
 * @apioption …
 */

(''); // keeps doclets above in transpiled file